examples/alloydb-chat-message-history-example/alloydb_chat_message_history_example.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/tmc/langchaingo/internal/alloydbutil"
+	"github.com/tmc/langchaingo/llms"
+	"github.com/tmc/langchaingo/memory/alloydb"
+)
+
+// getEnvVariables loads the necessary environment variables for the AlloyDB connection
+// and the chat message history creation.
+func getEnvVariables() (string, string, string, string, string, string, string, string, string, string) {
+	// Requires environment variable ALLOYDB_USERNAME to be set.
+	username := os.Getenv("ALLOYDB_USERNAME")
+	if username == "" {
+		log.Fatal("environment variable ALLOYDB_USERNAME is empty")
+	}
+	// Requires environment variable ALLOYDB_PASSWORD to be set.
+	password := os.Getenv("ALLOYDB_PASSWORD")
+	if password == "" {
+		log.Fatal("environment variable ALLOYDB_PASSWORD is empty")
+	}
+	// Requires environment variable ALLOYDB_DATABASE to be set.
+	database := os.Getenv("ALLOYDB_DATABASE")
+	if database == "" {
+		log.Fatal("environment variable ALLOYDB_DATABASE is empty")
+	}
+	// Requires environment variable PROJECT_ID to be set.
+	projectID := os.Getenv("PROJECT_ID")
+	if projectID == "" {
+		log.Fatal("environment variable PROJECT_ID is empty")
+	}
+	// Requires environment variable ALLOYDB_REGION to be set.
+	region := os.Getenv("ALLOYDB_REGION")
+	if region == "" {
+		log.Fatal("environment variable ALLOYDB_REGION is empty")
+	}
+	// Requires environment variable ALLOYDB_INSTANCE to be set.
+	instance := os.Getenv("ALLOYDB_INSTANCE")
+	if instance == "" {
+		log.Fatal("environment variable ALLOYDB_INSTANCE is empty")
+	}
+	// Requires environment variable ALLOYDB_CLUSTER to be set.
+	cluster := os.Getenv("ALLOYDB_CLUSTER")
+	if cluster == "" {
+		log.Fatal("environment variable ALLOYDB_CLUSTER is empty")
+	}
+	// Requires environment variable ALLOYDB_TABLE to be set.
+	tableName := os.Getenv("ALLOYDB_TABLE")
+	if tableName == "" {
+		log.Fatal("environment variable ALLOYDB_TABLE is empty")
+	}
+	// Requires environment variable ALLOYDB_SESSION_ID to be set.
+	sessionID := os.Getenv("ALLOYDB_SESSION_ID")
+	if sessionID == "" {
+		log.Fatal("environment variable ALLOYDB_SESSION_ID is empty")
+	}
+	// Requires environment variable ALLOYDB_SCHEMA to be set.
+	schemaName := os.Getenv("ALLOYDB_SCHEMA")
```

Comment:
Please remove and we will use the default schema value

Reply:
I removed it as a parameter and added an empty string to the InitChatHistoryTable as a parameter for it to use the default schema.